source/routing/raptor_api.cpp

```diff
@@ -922,6 +922,9 @@ get_stop_points( const type::EntryPoint &ep, const type::Data& data,
             }
         }
         LOG4CPLUS_DEBUG(logger, result.size() << " sp found for admin");
+    } else {
+        LOG4CPLUS_WARN(logger, "invalid entry point object " << ep.uri);
```

Comment:
with the tests below, I think we can give a better error message, saying that ep.type is not valid